pygmt/tests/test_solar.py

```diff
@@ -0,0 +1,86 @@
+"""
+Tests for solar.
+"""
+import datetime
+
+import pytest
+from pygmt import Figure
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers.testing import check_figures_equal
+
+
+@check_figures_equal()
+def test_solar_set_terminator_datetime_string():
+    """
+    Test passing the solar argument with the day_night terminator and a
+    datetime string.
+    """
+    fig_ref, fig_test = Figure(), Figure()
+    # Use single-character arguments for the reference image
+    fig_ref.solar(R="d", J="W0/15c", B="a", T="d+d1990-02-17T04:25:00")
```

Comment:
```suggestion
    fig_ref.solar(R="d", J="W0/15c", B="a", T="c+d1990-02-17T04:25:00")
```

Change terminator type to "civil".